src/wordlists.rs

```diff
@@ -1,6 +1,17 @@
 //! Built-in wordlists, system wordlist and utils for retrieving them.
+// #![cfg_attr(not(feature = "stable"), feature(proc_macro_hygiene))]
```

Comment:
@B0ney what's this for?

Reply:
You can remove it. That was there from my debugging and I forgot to remove it completely.

I came across a few issues trying to use ```include_flate```. If remember correctly, just adding ```include-flate = "0.1.4"``` to ```cargo.toml``` gave me errors because it, by default, used an unstable feature which required the nightly compiler. 

My initial (proposed) solution was putting this:
```rust
#![cfg_attr(not(feature = "stable"), feature(proc_macro_hygiene))]
```

But after going to the crate's repo, I found the solution was to just replace ```include-flate = "0.1.4"```  with:
```flate = { version ="0.1.4",  features=["stable"] }```

Reply:
Got it